libretro/libretro-mapper.c

```diff
@@ -23,12 +23,13 @@
 #include <time.h>
 #define usleep sys_timer_usleep
 
-void gettimeofday (struct timeval *tv, void *blah)
+int gettimeofday (struct timeval *tv, void *blah)
```

Comment:
Why is this required? Not that it matters if it does not break anything, just curious.

Reply:
if (gettimeofday (&tv, &tz) < 0)

Reply:
libretro/libretro-core.c: In function 'ftime':
libretro/libretro-core.c:5617: error: void value not ignored as it ought to be